web/tests/filter.test.cjs

```diff
@@ -962,6 +986,9 @@ test("predicate_basics", ({override}) => {
     predicate = get_predicate([["channels", "public"]]);
     assert.ok(predicate({}));
 
+    predicate = get_predicate([["channels", "archived"]]);
+    assert.ok(predicate({}));
```

Comment:
I don't understand why this is testing with an empty message object. Shouldn't you be checking with a message in an archived channel vs. an unarchived channel?